### ISSUE TITLE
Cannot reproduce result for coco

### ISSUE THREAD

**Issue by KaiWU5** [hikvision-research/SSOD#13]
Following readme construction, I only got AP 0.065 after training on Seed 1 Percent 1 Coco standard while 25.81 ± 0.28 (+16.76) is claimed in the paper. Here's my setting:

Dataset:
percent 1 seed Coco standard

Pretrained baseline: 
I use the train_gpu2.sh modification and only got AP less than 5%.

Training:
I use train_gpu8.sh without modification and only got AP 0.065 less than 10%

My guess is that the pretrained baseline performs low. Could you share your model and log of pretrained baseline and semi-supevised training to help use reproduce results?

**Comment by AceCoooool** [comment]
We use the provided scripts to run all the experiments. Could you provide the training log to help us find the problem?

**Comment by KaiWU5** [comment]
Thanks for reply. The log is below. From the log, I used the same config you provided with changing the data path and pre-trained path to my location. 

[Uploading 20221026_105431.log…](Training log)

**Comment by AceCoooool** [comment]
We use mmdet==2.10.0 in our experiments, and there may some difference in later version. We will find this out and update our code soon.

**Comment by wangshaishaiya** [comment]
> Following readme construction, I only got AP 0.065 after training on Seed 1 Percent 1 Coco standard while 25.81 ± 0.28 (+16.76) is claimed in the paper. Here's my setting:
> 
> Dataset: percent 1 seed 1 Coco standard
> 
> Pretrained baseline: I use the train_gpu2.sh modification and only got AP 0.0030 less than 1%. I trained twice and AP 0.003 is repeatable.
> 
> Training: I use train_gpu8.sh without modification and only got AP 0.065 less than 10%
> 
> My guess is that the pretrained baseline performs low. Could you share your model and log of pretrained baseline and semi-supevised training to help use reproduce results?

你好，我也遇见了这个问题，尝试多次，确定是环境的问题，重新配置环境就可以啦，
Ubuntu==20.2.4 cuda==11.0 Python==3.6.9 torch=1.7.0 torchvision==0.8.1 mmdet==2.10.0 mmcv-full==1.2.7 seaborn==0.11.0
